src/appsignal/cli/demo.py

```diff
@@ -19,26 +22,73 @@ def init_parser(parser: ArgumentParser) -> None:
         AppsignalCLICommand._push_api_key_argument(parser)
 
     def run(self) -> int:
-        # Prompt for all the required config in a sensible order
-        self._name  # noqa: B018
-        self._environment  # noqa: B018
-        self._push_api_key  # noqa: B018
-
-        client = Client(
-            active=True,
-            environment=self._environment,
-            name=self._name,
-            push_api_key=self._push_api_key,
-        )
+        cwd = os.getcwd()
+        app_config_path = os.path.join(cwd, "__appsignal__.py")
+        if os.path.exists(app_config_path):
+            loader = SourceFileLoader("appsignal", app_config_path)
+            spec = spec_from_loader(loader.name, loader)
+            if not spec:
+                # Not sure when this happens, but mypy complains that `spec`
+                # can be Optional.
+                print("Invalid __appsignal__.py config file found. Exiting.")
+                return 1
+
+            app_config_module = module_from_spec(spec)
+            loader.exec_module(app_config_module)
+            try:
+                client = app_config_module.appsignal
```

Comment:
Nit: I don't think there's anything wrong with this implementation, but an easier way to do this might be using [the `runpy` module](https://docs.python.org/3/library/runpy.html), which _I think_ uses the same mechanisms internally, but exposes them in a less piece-meal manner:

```python
from runpy import run_path
client = run_path(app_config_path)["appsignal"]
```

We already use this [in our Hatch build hooks](https://github.com/appsignal/appsignal-python/blob/main/src/scripts/build_hook.py#L17-L22), so there would be an advantage in not having to grok another way of doing the same thing.

Reply:
That's interesting. And runpy is just built in. Why couldn't I find a good resource for it then.